docs/Config_Changes.md

```diff
@@ -8,6 +8,10 @@ All dates in this document are approximate.
 
 ## Changes
 
+20240709: The `z_compensations` parameter in the `[axis_twist_compensation]`
+config section has been renamed to `zx_compensations`. If you don't want
+to recalibrate your x_axis_twist_compensation, simply rename the parameter.
```

Comment:
I would either maintain compatibility and print a warning about the setting being deprecated, or put a fatal error if that setting is present in the config file. At some point, I will upgrade my klipper and I will lose my calibration silently, this is not very user-friendly.

Reply:
I'm going to remove this; to make it fully backward compatible, I will keep z_compensations instead of using zx_compensations